src/app/[lng]/(infos)/layout.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FeatureCards } from '@/components/FeatureCards';
+import { ThemeProviderV2 } from '@/providers/ThemeProviderV2';
+import { Layout } from 'src/Layout';
+import { ThemeProvider } from 'next-themes';
+import { cookies } from 'next/headers';
+
+export default async function PartnerThemeLayout({
+  children,
+  params: { partnerTheme },
+}: {
+  children: React.ReactNode;
+  params: { partnerTheme: string };
+}) {
+  const cookies1 = cookies();
+
+  return (
+    <ThemeProvider
```

Comment:
let's rename this to `ThemeModeProvider`

Reply:
Hum, I disagree, this is very different than ThemeMode thing, I can rename it to NextThemeProvider to respect the 3rd party library name if you prefer